### PR TITLE
Add vasu copy to duplicates

### PR DESCRIPTION
**Pull request by patari**

### CODE REVIEW

File: service/src/integrationTest/kotlin/fi/espoo/evaka/vasu/VasuIntegrationTest.kt [hunk 1]
```diff
@@ -338,6 +341,15 @@ class VasuIntegrationTest : FullApplicationTest(resetDbBeforeEach = true) {
                 CurriculumType.PRESCHOOL -> ChildLanguage("kiina", "kiina")
             }
 
+        db.transaction { tx ->
```
Comment (zch) [inline service/src/integrationTest/kotlin/fi/espoo/evaka/vasu/VasuIntegrationTest.kt:344]: Should there be an assertion somewhere as well?

File: service/src/main/kotlin/fi/espoo/evaka/vasu/VasuController.kt [hunk 2]
```diff
@@ -389,12 +393,46 @@ class VasuController(
...
+                if (events.contains(MOVED_TO_CLOSED)) {
+                    val duplicates = tx.listPersonByDuplicateOf(document.basics.child.id)
+                    if (duplicates.isNotEmpty()) {
+                        val child = tx.getChild(document.basics.child.id)!!
+                        val template = tx.getVasuTemplate(document.templateId)!!
+                        val allEvents = document.events + addedEvents
+                        duplicates.forEach { duplicate ->
+                            val duplicateChild =
+                                tx.getChild(duplicate.id)
+                                    ?: tx.createChild(child.copy(id = duplicate.id))
```
Comment (zch) [inline service/src/main/kotlin/fi/espoo/evaka/vasu/VasuController.kt:414]: I'm probably missing something here, but why do we need to create a `child` for a duplicated person here? Are these not created when the person is duplicated?

Or is this just being defensive for a potential case of bad data?

File: service/src/integrationTest/kotlin/fi/espoo/evaka/vasu/VasuIntegrationTest.kt (hunk 1 above)
Comment (patari) [reply service/src/integrationTest/kotlin/fi/espoo/evaka/vasu/VasuIntegrationTest.kt]: Added!

File: service/src/main/kotlin/fi/espoo/evaka/vasu/VasuController.kt (hunk 2 above)
Comment (patari) [reply service/src/main/kotlin/fi/espoo/evaka/vasu/VasuController.kt:414]: The latter.

**Review by zch — APPROVED** [review]

**Pull request merged**